### ISSUE TITLE
[Bug]: Accessibility, a row in table should be focusable

### ISSUE THREAD

**Issue by lefuturiste** [navidrome/navidrome#2276]
### I confirm that:

- [X] I have searched the existing [open AND closed issues](https://github.com/navidrome/navidrome/issues?q=is%3Aissue) to see if an issue already exists for the bug I've encountered
- [X] I'm using the latest version (your issue may have been fixed already)

### Version

0.49.3

### Current Behavior

The row elements in most of the views (Album list view, Songs list view, etc) are not reachable when using tab.
When someone want to navigate using the Tab key they need to have this kind of element in reach to be able to navigate and play musics.

This issue can be seen by the inspector accessiblity developer tool in firefox:

![image](https://user-images.githubusercontent.com/20988163/227735854-9035d0d0-e5b6-40a2-8da6-cf28dd0efb45.png)

### Expected Behavior

The row element must be focusable and accessible when using <Tab> to navigate.

### Steps To Reproduce

1. Go to any Album List view in Table (rows) mode
2. Try to navigate using only <Tab> to any of the row.

### Environment

```markdown
- OS: Linux
- Browser: Firefox 112
- Client: Web UI
```

### How Navidrome is installed?

Docker

### Code of Conduct

- [X] I agree to follow Navidrome's Code of Conduct

**Comment by deluan** [comment]
Thanks for reporting. I'll be rewriting the UI using the latest ReactAdmin in an upcoming version, and will make sure this works as expected.
